bin

```diff
@@ -1,19 +1,42 @@
+#!/usr/bin/env node
+
 "use strict";
 
 const connect = require("connect");
-const fs = require("fs");
 const http = require("http");
+const fs = require("fs");
 const MemoryFS = require("memory-fs");
+const path = require("path");
 const webpack = require("webpack");
 
 const app = connect();
 const memoryFs = new MemoryFS();
 
 app.use(function(req, res) {
-    const path = req.url;
+    const requirePath = path.join(
```

Comment:
This is so I can `require()` files anywhere in the filesystem from `entry.js`, which will be located maybe in `/usr/bin/local`? It's not working, though, perhaps I won't be able to `require()` dynamically with webpack like I want to.